test/cli/run/glob-on-fuse.test.ts

```
@@ -0,0 +1,124 @@
+          // unmount
+          const umount = spawn({ cmd: ["fusermount", "-u", mountpoint] });
+          await umount.exited;
+          // wait for graceful exit
+          await Promise.race([pythonProcess.exited, Bun.sleep(1000)]);
+          expect(pythonProcess.exitCode).toBe(0);
+        } catch (e) {
+          pythonProcess.kill("SIGKILL");
+          console.error("python process errored:", await new Response(pythonProcess.stderr).text());
+          // Don't re-throw cleanup errors to preserve original test failures
+          if (!originalError) {
+            throw e;
```

Biome (2.1.2)

[error] Unsafe usage of 'throw'.

'throw' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)